frontend/src/components/Utils/utils.test.js

```diff
@@ -0,0 +1,37 @@
+import * as utils from './utils';
```

Comment:
Include '.js' suffix as in Hiram's comment on keiffer's PR #40. May want to do a scan over all other files for this too.

Reply:
Fixed in a future branch :P

Reply:
A global fix PR would be fine, but I encourage changing it here since this isn't submitted yet. I will let you make the final judgement call, though.

Reply:
#46